.travis.yml

```diff
@@ -8,13 +8,15 @@ env:
   matrix:
   - BUILDTOOL=autotools
   - BUILDTOOL=cmake
-  - BUILDTOOL=autotools_old_compilers
```

Comment:
OK, I'll reverse that... O:-)

Regarding the old compilers thing, they can be just targzipped an left in any URL (e.g. as a fake github release), then just downloaded and uncompressed in the working directory.